src/content/ui-api/management-api/authentication/index.md

```diff
@@ -121,13 +121,89 @@ Let's go into details for the most relevant properties (also called "claims") of
 
-Each Giant Swarm installation provides a service account named `automation` in the `default` namespace.
+### Remote access to the Management Cluster {#remote-mc}
 
-**Note:** This service account comes with a quite powerful set permissions. **We strongly recommend to create a specific service account for each application**, binding it to specific roles granting only the required permissions in the required namespaces.
+Each Giant Swarm installation provides a service account named `automation` in the `default` namespace that may be used for creating a self-contained `kubeconfig` file. **Note however**, this account comes with a powerful set of permissions, thus **we strongly recommend you create a specific service account for each application**, binding it to specific roles granting only the required permissions in the required namespaces.
+
+Regardless of which service account you decide to use. From the management cluster, you can utilize the step below to create a config file.
+
+1. Export the Service Account name:
+
+```nohighlight
+SA_NAME=automation
+```
+
+2. Find Kubernetes Secret associated with the service account:
+
+```nohighlight
+SECRET=$(kubectl get sa $SA_NAME -o jsonpath='{.secrets[0].name}')
+```
+
+3. Find the service account's token:
+
+```nohighlight
+TOKEN=$(kubectl get secret $SECRET -o jsonpath='{.data.token}' | base64 --decode)
+```
+
+4. Find Kubernetes CA certificate:
+
+```nohighlight
+CA_CERT=$(kubectl get secret $SECRET -o jsonpath='{.data.ca\.crt}')
+```
+
+5. Grab installation's API endpoint:
+
+```nohighlight
+API_URL=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
+```
+
+6. Grab installation's name:
+
+```nohighlight
+MC_NAME=$(kubectl config view --minify -o jsonpath='{.clusters[0].name}')
+```
+
+7. Generate kubectl configuration:
+
+```nohighlight
+cat <<EOF > kubeconfig
+apiVersion: v1
+kind: Config
+clusters:
+  - name: $MC_NAME
+    cluster:
+      certificate-authority-data: $CA_CERT
+      server: $API_URL
+users:
+  - name: $SA_NAME
+    user:
+      token: $TOKEN
+current-context: $MC_NAME
+contexts:
+- context:
+    cluster: $MC_NAME
+    user: $SA_NAME
+  name: $MC_NAME
+EOF
+```
+
+8. Test newly created file:
+
+```nohighlight
+kubectl --kubeconfig kubeconfig cluster-info
+```
+
+### Remote access to the Workload Cluster {#remote-wc}
+
+Workload Clusters rely on the [TokenRequest API](https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/token-request-v1/#create-create-token-of-a-serviceaccount) for authentication, so the steps from the previous paragraph will not work against them. We strongly encourage you to use the `kubectl-gs` plugin and [client certificate login with a self-contained kubeconfig](https://docs.giantswarm.io/ui-api/kubectl-gs/login/#wc-client-cert-automation) instead.
```

Comment:
For links within docs, we use `relref` which is a HUGO shortcode that helps ensure link integrity, as it fails on build time if the target is not found.

```suggestion
Workload clusters rely on the [TokenRequest API](https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/token-request-v1/#create-create-token-of-a-serviceaccount) for authentication, so the steps from the previous paragraph will not work against them. We strongly encourage you to use the `kubectl-gs` plugin and [client certificate login with a self-contained kubeconfig]({{< relref "/ui-api/kubectl-gs/login/#wc-client-cert-automation" >}}) instead.
```